data/team.js

```diff
@@ -0,0 +1,53 @@
+export const team = {
+   
+    list: [
+      {
+        photo: "./images/cto.png",
+        function: 'CTO',
+        data: 'Johny B Good',
+        description: 'The brains behind the whole operation',
+        icons: [
+            './images/team-twitter.svg',
+            './images/team-facebook.svg'
```

Comment:
Często przy wykorzystaniu webpacka pliki graficzne się importuje (https://webpack.js.org/guides/asset-management/#loading-images), aby webpack mógł je optymalizować.

PS. Na chwilę obecną u mnie grafiki się nie wczytują...
![image](https://github.com/devmentor-pl/task-html-and-css-basics/assets/55095257/cc11ac87-1fa9-4e91-a60e-f3af2883fa4c)